tracer/src/Datadog.Trace/Iast/Settings/IastSettings.cs

```diff
@@ -34,6 +34,11 @@ internal class IastSettings
     /// </summary>
     internal const string DefaultRedactionValuesRegex = @"(?i)(?:bearer\s+[a-z0-9\._\-]+|glpat-[\w\-]{20}|gh[opsu]_[0-9a-zA-Z]{36}|ey[I-L][\w=\-]+\.ey[I-L][\w=\-]+(?:\.[\w.+/=\-]+)?|(?:[\-]{5}BEGIN[a-z\s]+PRIVATE\sKEY[\-]{5}[^\-]+[\-]{5}END[a-z\s]+PRIVATE\sKEY[\-]{5}|ssh-rsa\s*[a-z0-9/\.+]{100,})|[\w\.-]+@[a-zA-Z\d\.-]+\.[a-zA-Z]{2,})";
 
+    /// <summary>
+    /// Default values readaction regex if none specified via env DD_IAST_REDACTION_VALUES_REGEXP
+    /// </summary>
+    internal const string DefaultCookieFilterRegex = @".{32,}";
```

Comment:
This regex is "any string of 32+ characters, right?" Does that catch all the cookies we were previously filtering, i.e. `AspNetCore` cookies?

Reply:
Those are ignored, not filtered. It's a different case. Filtered cookies are reported, but share the same hash, so they should be grouped in backed.

Reply:
Ahhhh, ok, got it, thanks!